docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,452 @@
 # Quick Start
 
-## Try Nebula Graph from Docker
+This guide helps you walk through the process of using **Nebula Graph**. We will show you how to [create and use a graph space](#creating-and-using-a-graph-space), [define the schema for your data](#defining-the-schema-for-your-data), [insert data](#inserting-data), [fetch data](#fetching-data), [update data](#updating-data) and [delete data](#deleting-data). At the end we will show you how to [insert multiple data](#batch-inserting) with a `.ngql` file.
 
-The easiest way to get Nebula Graph up and running is using Docker. Before you start, make sure that you have:
+## Overview
 
-* Installed the latest version of [Docker](https://docs.docker.com/)
+We will show you how to use the **Nebula Graph** database based on the relations between different vertices in the following figure:
 
-* Pulled the latest images of Nebula from [Nebula Docker Hub](https://hub.docker.com/r/vesoft/nebula-graph). If not, pull the images using the following command:
+![image](https://user-images.githubusercontent.com/40747875/69215979-fd92ff00-0ba5-11ea-9bd4-31841b5a2a5f.png)
 
-```bash
-> docker pull vesoft/nebula-graph:nightly
-```
+In the above figure, there are two tags(**player**, **team**) and two edge types (**serve** and **follow**).
```

Comment:
**tag**
**_edge_**

Reply:
As we discussed, both should be in Bold. Generally, for variables, italic is the better choice.